.gitignore

```diff
@@ -13,3 +13,4 @@
 .vscode
 .coverage*
 .idea
+/signingkeyserver.db
```

Comment:
Mmm, what generates this file? Also ideally this change would be in a separate PR :slightly_smiling_face:

Reply:
Dendrite does.

Does it happen because this file is not yet overridden in `Dendrite.pm` to Postgres like other files are? Should `*.db` be in `.gitignore` instead for the case when the tests are run without the `POSTGRES` variable?